### PR TITLE
[Done] Translation of meetups.md ..

### PR DESCRIPTION
**Pull request by 3imed-jaberi**
~All REACT NATIVE meetups have been deleted because we are in REACTJS site .~
~All REACT NATIVE meetups must be in his site.~

Has been fixed ..

### CODE REVIEW

**Comment by Fcmam5** [comment]
Meetup(s) is better translated to "مُلْتَقى (مُلْتَقَيات)". It's not a working session (جلسة عمل).
Actually, React meetups are ones of the most fun tech-meetups with just a few talks/workshops and food and fun :smiley:

**Comment by 3imed-jaberi** [comment]
> Meetup(s) is better translated to "مُلْتَقى (مُلْتَقَيات)". It's not a working session (جلسة عمل).
> Actually, React meetups are ones of the most fun tech-meetups with just a few talks/workshops and food and fun 😃 .
> Also, we must be "fidèle" to the official documentation and keep it as it is, and to keep RN (ReactNative) meetups links because it's a React-Team decision to keep it there, if you think that it should be there, you can file an issue in the #reactjs/reactjs.org repo and ask them to move it to RN website and make a simple link to it in that page.

@Fcmam5 
Thanks bro , 
No problem I was just talking to Ryan at Slack about this ....

File: content/community/meetups.md [hunk 1]
```diff
@@ -1,152 +1,147 @@
...
+## بيرو {#peru}
+* [ليما](https://www.meetup.com/ReactJS-Peru/)
+
+## فرنسا {#france}
+* [باريس](https://www.meetup.com/ReactJS-Paris/)
+
+## كندا {#canada}
+* [أوتاوا](https://www.meetup.com/Ottawa-ReactJS-Meetup/)
+* [فانكوفر](https://www.meetup.com/ReactJS-Vancouver-Meetup/)
+* [مونتريال](https://www.meetup.com/fr-FR/ReactMontreal/)
```
Comment (Fcmam5) [inline content/community/meetups.md]: Missing link:

```suggestion
* [مونتريال](https://www.meetup.com/fr-FR/ReactMontreal/)
* [مونتريال - React Native](https://www.meetup.com/fr-FR/React-Native-MTL)
```

File: content/community/meetups.md [hunk 2]
```diff
@@ -1,152 +1,147 @@
...
+* [جايبور](https://www.meetup.com/JaipurJS-Developer-Meetup/)
+  
+## الولايات المتحدة {#us}
+* [أتلانتا (جورجيا)](https://www.meetup.com/React-ATL/)
+* [أرفاين (كالفورنيا)](https://www.meetup.com/ReactJS-OC/)
+* [أوستن (تكساس)](https://www.meetup.com/ReactJS-Austin-Meetup/)
+* [بالو ألتو (كاليفورنيا) - React Native](https://www.meetup.com/React-Native-Silicon-Valley/)
+* [بروفو (يوتا)](https://www.meetup.com/ReactJS-Utah/)
+* [بورتلاند (أوريغون)](https://www.meetup.com/Portland-ReactJS/)
+* [بوسطن (خليج ماساشوستس)](https://www.meetup.com/ReactJS-Boston/)
```
Comment (Fcmam5) [inline content/community/meetups.md]: Missing link:
```suggestion
* [بوسطن (خليج ماساشوستس)](https://www.meetup.com/ReactJS-Boston/)
* [بوسطن (خليج ماساشوستس) - React Native](https://www.meetup.com/Boston-React-Native-Meetup/)
```

File: content/community/meetups.md [hunk 3]
```diff
@@ -1,152 +1,147 @@
...
+* [بيتسبرغ (بنسيلفانيا)](https://www.meetup.com/ReactPgh/)
+* [تامبا (فلوريدا)](https://www.meetup.com/ReactJS-Tampa-Bay/)
+* [توسان (أريزونا)](https://www.meetup.com/Tucson-ReactJS-Meetup/)  
+* [دالاس (تكساس)](https://www.meetup.com/ReactDallas/)
+* [ساكرامنتو (كاليفورنيا)](https://www.meetup.com/Sacramento-ReactJS-Meetup/)
+* [سانتا مونيكا (كاليفورنيا)](https://www.meetup.com/Los-Angeles-ReactJS-User-Group/)
+* [سان فرانسيسكو (سان فرانسيسكو) - React JS](https://www.meetup.com/ReactJS-San-Francisco/)
+* [سان فرانسيسكو (سان فرانسيسكو) - React Native](https://www.meetup.com/React-Native-San-Francisco/)
```
Comment (Fcmam5) [inline content/community/meetups.md]: Missing link

```suggestion
* [سان فرانسيسكو (سان فرانسيسكو) - React Native](https://www.meetup.com/React-Native-San-Francisco/)
* [سان فرانسيسكو (سان فرانسيسكو) - React في العالم الحقيقي (Real World React)](https://www.meetup.com/Real-World-React)
```

or:
```suggestion
* [سان فرانسيسكو (سان فرانسيسكو) - React Native](https://www.meetup.com/React-Native-San-Francisco/)
* [سان فرانسيسكو (سان فرانسيسكو) - Real World React](https://www.meetup.com/Real-World-React)
```

File: content/community/meetups.md [hunk 4]
```diff
@@ -1,152 +1,147 @@
...
+* [أوستن (تكساس)](https://www.meetup.com/ReactJS-Austin-Meetup/)
+* [بالو ألتو (كاليفورنيا) - React Native](https://www.meetup.com/React-Native-Silicon-Valley/)
+* [بروفو (يوتا)](https://www.meetup.com/ReactJS-Utah/)
+* [بورتلاند (أوريغون)](https://www.meetup.com/Portland-ReactJS/)
+* [بوسطن (خليج ماساشوستس)](https://www.meetup.com/ReactJS-Boston/)
+* [بيتسبرغ (بنسيلفانيا)](https://www.meetup.com/ReactPgh/)
+* [تامبا (فلوريدا)](https://www.meetup.com/ReactJS-Tampa-Bay/)
+* [توسان (أريزونا)](https://www.meetup.com/Tucson-ReactJS-Meetup/)  
+* [دالاس (تكساس)](https://www.meetup.com/ReactDallas/)
+* [ساكرامنتو (كاليفورنيا)](https://www.meetup.com/Sacramento-ReactJS-Meetup/)
```
Comment (Fcmam5) [inline content/community/meetups.md]: Missing link:

```suggestion
* [سان رامون (كاليفورنيا) - مطوري تري-فالي (TriValley Coders)](https://www.meetup.com/trivalleycoders/)
* [ساكرامنتو (كاليفورنيا)](https://www.meetup.com/Sacramento-ReactJS-Meetup/)
```

or


```suggestion
* [سان رامون (كاليفورنيا) - TriValley Coders](https://www.meetup.com/trivalleycoders/)
* [ساكرامنتو (كاليفورنيا)](https://www.meetup.com/Sacramento-ReactJS-Meetup/)
```

**Pull request merged**